securedrop/requirements/python3/develop-requirements.in

```diff
@@ -18,6 +18,7 @@ molecule-vagrant>=0.3
 # Needed for ansible network filter
 # http://docs.ansible.com/ansible/latest/playbooks_filters_ipaddr.html
 netaddr
+peewee>=3.17.1  # Safety 64952
```

Comment:
Since you've already updated this in the requirements.txt, we can remove it from requirements.in so it doesn't look like a dependency we actually want rather than a transitive dependency.

Reply:
Good call.  Amended in 2b8761ca9c5eab408a537654599e4f1b448dbd47.